sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityPropertyToTConverterFactoryTests.cs

```diff
@@ -0,0 +1,907 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host.Converters;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.Cosmos.Table;
+using Microsoft.Azure.WebJobs.Extensions.Tables;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Tables.Tests
+{
+    public class EntityPropertyToTConverterFactoryTests
+    {
+        [Test]
+        public void Create_EntityProperty_CanConvert()
+        {
+            // Act
+            IConverter<EntityProperty, EntityProperty> converter =
+                EntityPropertyToTConverterFactory.Create<EntityProperty>();
+            // Assert
+            Assert.NotNull(converter);
+            EntityProperty expected = new EntityProperty(1);
+            EntityProperty property = converter.Convert(expected);
+            Assert.AreEqual(expected, property);
+        }
+
+        [Theory]
+        [TestCase(false)]
+        [TestCase(true)]
+        public void Create_Boolean_CanConvert(bool expected)
+        {
+            // Act
+            IConverter<EntityProperty, bool> converter = EntityPropertyToTConverterFactory.Create<bool>();
+            // Assert
+            Assert.NotNull(converter);
+            EntityProperty property = new EntityProperty(expected);
+            bool actual = converter.Convert(property);
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        public void Create_Boolean_ConvertThrowsIfNullProperty()
```

Comment:
nit: these could all be turned into 4 or 5 tests with different types fed to the TestCaseSource.

Reply:
I didn't change this file. I initially removed it and then added it back.